guides/13_Project_Board.md

```diff
@@ -0,0 +1,22 @@
+# Amazon Chime SDK Project Board
+
+The [Amazon Chime SDK Project Board](https://github.com/orgs/aws/projects/12) provides builders with an overview of community feature requests and their status across all the repositories for the Amazon Chime SDK ([Amazon Chime SDK for JavaScript](https://github.com/aws/amazon-chime-sdk-js), [Amazon Chime SDK React Component Library](https://github.com/aws/amazon-chime-sdk-component-library-react), [Amazon Chime SDK for iOS](https://github.com/aws/amazon-chime-sdk-ios) and [Amazon Chime SDK for Android](https://github.com/aws/amazon-chime-sdk-android)). Each repository has a [Community Issue template](https://github.com/amazon-chime-sdk-js/.github/ISSUE_TEMPLATE/community-request-template.md) that allows builders to contribute community feature ideas for the Amazon Chime SDK.
+
+The Amazon Chime team would like to prioritize issues that are most useful for our developer community.  We recommend you give us a thumbs-up to the features on our Project Board that look interesting to you. 
+
+Note: This Project Board is provided for informational purposes only. AWS does not commit to release any solution or feature described in the Project Board. 
+
+## ****Project Board Categories****
+
+Here is what each of the different categories in the Project Board mean:
+
+**Tell Us What You Need:** This is a list of feature requests that community members of Amazon Chime SDK have expressed their interest in. Features in this category have not yet been evaluated by the Amazon Chime SDK team.
+
+**Researching:** We are considering it, but cannot commit if, or when, we will work on items in this list. This means we are evaluating the feature and how it might work, to determine whether it is something we may want to start working on. We may propose mitigations or alternative options (if they exist) and encourage our community members to also make suggestions that would interest the community.
+
+**We’re working on it:** This is work in progress and we are evaluating design decisions or working through implementation, automation, testing, approvals, documentation and release mechanics. Items in this column may stay here for several months as we work through the implementation.
+
+**Shipped:** This is a launched feature that is supported by AWS to be used in your production workloads. You will find links to the Amazon Chime SDK version containing this feature and any supporting documentation.
+
+Choose [this link](https://github.com/orgs/aws/projects/12) to see projects that Amazon Chime is currently working on.
```

Comment:
Should we say "Click" or "Bookmark" here in place of "Choose"?

Reply:
CHoose is a standard term in public documentation - so i will keep this